app-config.yaml

```diff
@@ -52,31 +52,31 @@ proxy:
 # Note: After experimenting with basic setup, use CI/CD to generate docs
 # and an external cloud storage when deploying TechDocs for production use-case.
 # https://backstage.io/docs/features/techdocs/how-to-guides#how-to-migrate-from-techdocs-basic-to-recommended-deployment-approach
-techdocs:
-  builder: 'external'
-  publisher:
-    type: 'awsS3'
-    awsS3:
-      bucketName: ${AWS_BUCKET_NAME}
-      region: ${AWS_BUCKET_REGION}
-      credentials:
-        accessKeyId: ${AWS_ACCESS_KEY_ID}
-        secretAccessKey: ${AWS_ACCESS_KEY_SECRET}
-
 # techdocs:
-#   builder: 'local'
+#   builder: 'external'
 #   publisher:
-#     type: 'local'
-#   generator:
-#     techdocs: local
+#     type: 'awsS3'
+#     awsS3:
+#       bucketName: ${BIH_AWS_BUCKET_NAME}
+#       region: ${BIH_AWS_BUCKET_REGION}
+#       credentials:
+#         accessKeyId: ${AWS_ACCESS_KEY_ID}
+#         secretAccessKey: ${AWS_ACCESS_KEY_SECRET}
+
+techdocs:
```

Comment:
If this change is permanent then does this documentation need to be altered to accompany it? 

https://github.com/department-of-veterans-affairs/bih/blob/develop/README.md